README.md

```diff
@@ -85,106 +89,95 @@ The Realm Flutter package name is `realm`
 * Supported platforms are Flutter (iOS, Android, Windows, MacOS) and Dart standalone (Windows, MacOS and Linux)
 
 * Flutter ^2.8.0
+* For Flutter Desktop environment setup check the guide [here](https://docs.flutter.dev/desktop)
 
 ## Usage
 
 * Add `realm` package to a Flutter application.
+
     ```
     flutter pub add realm
     ```
 
-* Enable generation of RealmObjects.
-
-    * Add `build_runner` package to `dev_dependencies`.
-        ```
-        dart pub add build_runner --dev
-        ```
-    * Enable `realm_generator` by adding a `build.yaml` file to the application.
-        ```yaml
-        targets:
-            $default:
-                builders:
-                    realm_generator|realm_object_builder:
-                        enabled: true
-                        generate_for:
-                            - lib/*.dart 
-        ```
-
-    * Import Realm in a dart file (ex. `catalog.dart`).
-        ```dart
-        import 'package:realm/realm.dart';
-        ```
-
-    * Declare a part file `catalog.g.dart` in the begining of the `catalog.dart` dart file after all imports.
-
-        ```dart
-        import 'dart:io';
-
-        part 'catalog.g.dart'
-        ```
-
-    * Create a data model class
-
-        ```dart
-        @RealmModel()
-        class _Item {
-            @PrimaryKey()
-            late final int id;
-
-            late String name;
-            
-            int price = 42;
-        }
-        ```
-    * Generate RealmObject class `Item` from data model class `_Item`.
-
-        ```
-        dart run realm generate
-        ```
-        A new file `catalog.g.dart` will be created next to the `catalog.dart`.
-        
-        _*This file should be committed to source control_
+* Import Realm in a dart file (ex. `catalog.dart`).
 
-    * Use the RealmObject class `Item` with Realm.
+    ```dart
+    import 'package:realm/realm.dart';
+    ```
 
-        ```dart
-        // Create a Configuration object
-        var config = Configuration(Item.schema);
+* Declare a part file `catalog.g.dart` in the begining of the `catalog.dart` dart file after all imports.
 
-        // Opean a Realm
-        realm = Realm(config);
+    ```dart
+    import 'dart:io';
 
-        // Open a write transaction
-        realm.write(() {
-          var myItem = Item(0, 'Iteam', price: 4);
-          realm.add(myItem);
-          final item = realm.add(Item(1, 'Iteam')..price = 20);
-        });
+    part 'catalog.g.dart'
+    ```
 
-        // Get objects from the realm
+* Create a data model class.
 
-        // Get all objects
-        var items = realm.all<Item>();
-        
-        // Get object by index
-        var item = items[5];
-        
-        // Get object by primary key
-        var itemByKey = realm.find<Item>(0);
+    It should start with an underscore `_Item` and be annotated with `@RealmModel()`
```

Comment:
or `$Item`

Reply:
This should be better documented in the docs.